tests/slow/src/test/scala/tests/feature/DefinitionCrossLspSuite.scala

```diff
@@ -19,6 +19,36 @@ class DefinitionCrossLspSuite
     }
   }
 
+  test("underscore") {
+    cleanDatabase()
+    for {
+      _ <- server.initialize(
+        s"""
+           |/metals.json
+           |{
+           |  "a": {
+           |    "scalaVersion": "${BuildInfo.scala213}"
```

Comment:
This is unrelated to switching to default scala 2.13, correct?
Or you wanted to test that default scala would be 2.13 by veryfing 2.13 only feature would be parsed?

Reply:
That is related, I wanted to make sure we properly parse 2.13 syntax.